templates/stamp/stamp-v2.json

```diff
@@ -0,0 +1,1174 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
Looks like you're defining the same parameters in templates/stamp/stamp-keyvault.json.  Is there any means to externalize these and reference the json file (e.g templateLink)?